kernel/src/memory/vm/include/swap_fs.h

```diff
@@ -37,7 +37,7 @@ int64_t swapfs_store(void*, uint32_t*, uint8_t);
 
 
 /* swapfs_retrieve will retrieve the page identified by the ID pointer and
- * will store it back into main memory (specified by the void *page pointer)
+ * will store it back into common memory (specified by the void *page pointer)
```

Comment:
it seems you have mass renamed "main" -> "common", I don't know if it was intentional to edit comments such as this